CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Unreleased changes
 
+* General
+  * BUGFIX: fix `int32` to `ErrorType` mapping. ([#1063](https://github.com/mozilla/glean/pull/1063))
```

Comment:
```suggestion
  * BUGFIX: fix `int32` to `ErrorType` mapping. The `InvalidOverflow` had a value mismatch between glean-core and the bindings. This would only be a problem in unit tests. ([#1063](https://github.com/mozilla/glean/pull/1063))
```